technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note.csl

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Finanzwirtschaft und Finanzdienstleistungen (note)</title>
+    <title-short>TUD Betriebswirtschaftslehre (note)</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note" rel="self"/>
```

Comment:
Does the Lehrstuhl document that style online somewhere? We really like to have links to documentation where at all possible. If so, include as `rel="documentation"` link.